src/ONYXKEYS.ts

```diff
@@ -427,6 +427,15 @@ const ONYXKEYS = {
         // Shared NVPs
         /** Collection of objects where each object represents the owner of the workspace that is past due billing AND the user is a member of. */
         SHARED_NVP_PRIVATE_USER_BILLING_GRACE_PERIOD_END: 'sharedNVP_private_billingGracePeriodEnd_',
+
+        /** Expensify cards settings */
+        SHARED_NVP_PRIVATE_EXPENSIFY_CARD_SETTINGS: 'sharedNVP_private_expensifyCardSettings_',
+
+        /**
+         * Stores the card list for a given fundID and feed in the format: card_<fundID>_<bankName>
+         * So for example: card_12345_ExpensifyCard
+         */
+        EXPENSIFY_CARDS_LIST: 'card_',
```

Comment:
Fair point. I'm thinking `EXPENISFY_CARDS_LIST` still makes the most sense.

Alternatively, we can go with `WORKSPACE_CARDS_LIST` and then for domain feeds it could a new const called `DOMAIN_CARDS_LIST`

Reply:
But having just one makes the most sense, I think so I'm still fine with  `EXPENISFY_CARDS_LIST`

Reply:
Okay, looks like we're going with `WORKSPACE_CARDS_LIST`. I'm fine with that since it is more clear. We can change it down the line without much difficulty if we find it's not suitable for domain feeds.